ixmp/tests/core/test_meta.py

```diff
@@ -0,0 +1,67 @@
+"""Test meta functionality of ixmp.Platform."""
+
+import pytest
+
+import ixmp
+from ixmp.testing import models
+
+
+sample_meta = {'sample_string': 3, 'another_string': 'string_value'}
```

Comment:
first entry has not a string value. What is the idea? Maybe just called by mistake